main.go

```diff
@@ -1398,6 +1396,7 @@ func main() {
 	ocdCommandsString := flag.String("ocd-commands", "", "OpenOCD commands, overriding target spec (can specify multiple separated by commas)")
 	ocdOutput := flag.Bool("ocd-output", false, "print OCD daemon output during debug")
 	port := flag.String("port", "", "flash port (can specify multiple candidates separated by commas)")
+	maxRetries := flag.Int("retries", 10, "the maximum number of times to retry locating the MSD volume to be used for flashing")
```

Comment:
IMHO, "retries" is very generic, shall have "msd" in the name? "msd-retries" or something like it?

Reply:
Perhaps `flash-retries` since it specifically has to do with the `tinygo flash` command?

```
tinygo flash -target=pico -flash-retries=15 -port=/dev/ttyACM0 ./path.to/code
```

Reply:
Hmm, for some reason `retries` just reads better to me:

```
tinygo flash -target=pico -retries=15 -port=/dev/ttyACM0 ./path.to/code
```

Probably because the word "flash" already appears once. Seems like stuttering.

Reply:
If it does work only for flash command, then yes, “retries” shall be ok actually.
When “retries” specified for, say, build or debug command, it is just ignored or an error happens?

Reply:
The flag is just ignored for those commands.